src/templates/_macros/app_tile.html

```diff
@@ -45,11 +46,13 @@
           {{ rating_link(is_detail=is_detail) }}
         {% endif %}
 
-        {{ market_button(app, data_attrs={
-               'manifest_url': app.manifest_url,
-               'source': src,
-               'slug': app.slug
-           }) }}
+        {% if not (app.isWebsite and is_detail) %}
```

Comment:
I wanted this to be the opposite of the `if`s below. And those are `if is_detail { if app.isWebsite }`.